x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -35,4 +35,45 @@ export function registerIntegrations(
     shipper: 'other',
     isBeta: false,
   });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_geojson',
+    title: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationTitle', {
+      defaultMessage: 'GeoJson',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationDescription', {
+      defaultMessage: 'Upload GeoJson with Maps.',
```

Comment:
There should be a noun after GeoJSON:

> Upload a GeoJSON file with Elastic Maps.

Or instead of repeating GeoJSON, can you add a few words that explain it.  Something like:

> Upload geographic data structures with Elastic Maps.

I'm suggesting "Elastic Maps" here because that'd how I see it on the website and I think it is more clear than simply "Maps".

Reply:
I like "Upload a GeoJSON files with Elastic Maps."

I added a plural to file since you can upload more than one. Does that sound ok?

> Upload geographic data structures with Elastic Maps.

This is a little vague since "geographic data structures" can mean lots of things.

Reply:
Got it.  Let's use:

> Upload GeoJSON files with Elastic Maps.